numba/core/errors.py

```diff
@@ -433,11 +433,17 @@ def deprecate_moved_module_getattr(old_module, new_module):
     locations
     """
     def wrapper(attr):
-        mod = import_module(new_module)
-        ret_attr = getattr(mod, attr)
-        msg = _moved_msg2.format(attr, old_module, new_module)
-        warnings.warn(msg, category=NumbaDeprecationWarning, stacklevel=2)
-        return ret_attr
+        if new_module is None:
+            msg = _moved_no_replacement
```

Comment:
Is it worth attempting to import `attr` here first?

Reply:
I'm pointing out that `import numba.target; x = numba.target.imputils` fails in .49 but works in 0.48.

Reply:
But could be made to work if, before giving up and raising, you attempt to import the submodule with the provided name.

Reply:
I've expanded on that in https://github.com/numba/numba/pull/5497